apps/docs/content/components/collections/TagGroup.mdx

```diff
@@ -157,6 +158,12 @@ Tags can have a loading state using the `isLoading` prop.
 
 <Example src="tag/docs/tag/loading" />
 
+### Avatar
+
+A tag can have an avatar.
```

Comment:
Should we always use the same sentence? Either A XYZ can have an avatar or A XYZ can contain an avatar.

Reply:
sure, which one do you want? I think contain makes more sense.